package.json

```diff
@@ -14,7 +14,8 @@
     "sync-up": "./scripts/sync-up",
     "upload-test-data": "./scripts/upload-test-data",
     "restore-test-data": "./scripts/restore-test-data",
-    "update": "node download && node massage"
+    "update": "node download && node massage",
```

Comment:
Small request, would you please merge current `main` into this branch and resolve the minor conflicts caused by my having also added these two under slightly different names? Then I can approve and we can merge. Thanks!